modules/util/helpers.js

```diff
@@ -24,3 +24,24 @@ export function createDeferred() {
 export function createTaskQueue() {
     return new TaskQueue();
 }
+
+/**
+ * Downloads a JSON object.
+ *
+ * @param json the JSON object to download.
+ * @param filename the filename to give to the downloaded file.
+ */
+export function downloadJSON(json, filename) {
```

Comment:
Maybe you can put the new function somewhere in the react folder? How about react/features/base/util?